packages/page-pattern-modal/src/styles/page-pattern-modal.scss

```diff
@@ -53,15 +65,20 @@ $sidebar-width-desktop: 324px;
 	.components-modal__content {
 		overflow-y: scroll;
 		-webkit-overflow-scrolling: touch;
-		// prevents a 1px strip of content being visible after scrolling
-		margin-top: -1px;
+		margin-top: 60px;
 		padding: 0 16px 16px;
+
+		&::before {
+			margin-bottom: 0;
+		}
 	}
 
 	.components-modal__header {
 		border-bottom: none;
 		margin-right: -16px;
+		background-color: #fff;
```

Comment:
Why is this needed, isn't it already white? I think erring on the side of being verbose with comments will help our future selves when we're back here again after a future Gutenberg release 🙂

Reply:
Yup already removed. This PR started out as a Jackson Pollock, now slowly moving towards a Mondrian 😅